docs/getting-started.md

```diff
@@ -144,6 +144,27 @@ Experiment tracker
 
 By default, Matcha will hide sensitive resource properties. If you need one of these properties, then you can add the `--show-sensitive` flag to your `get` command.
 
+# &#129309; Sharing Resources
+
+You'll notice that a configuration file is create as part of the provisioning process - it's called `matcha.config.json`. This file stores the information necessary for Matcha to link up to your provisioned resources.
+
+To share these resources, you need to add this file to your project repository (GitHub or similar).
```

Comment:
GitHub isn't strictly necessary, though I imagine it will be most common. I think there's a better logical flow if you also bring the next sentence forward. Something like;

"In order to access your provisioned resources, other users will need to ensure this configuration file exists locally, in the same directory where the file was originally created. We suggest that the matcha.config.json file be included within the project repository.